app/script/telemetry/calling/CallLogger.js

```diff
@@ -0,0 +1,186 @@
+z.telemetry.calling.CallLog = [];
```

Comment:
This array should preferably be initiated within the CallTelemetry instance and passed as an argument into the constructor.

Reply:
Not sure this would work, it needs to be accessible and shared among all other classes where CallLogger is initialized.